test/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:centos7
+
+LABEL name="EL7 OSG 3.5 Gratia Probe Test Container"
+
+RUN cd \
+    && yum install -y yum-priorities \
+    && yum install -y \
+       http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+    && yum install -y \
+       http://repo.opensciencegrid.org/osg/3.5/osg-3.5-el7-release-latest.rpm \
+    && yum install -y --enablerepo=devops osg-build \
+    && yum install -y --enablerepo=osg-empty slurm \
+    && git clone https://github.com/opensciencegrid/gratia-probe \
+    && ver=$(awk '/Version:/ {print $2}' gratia-probe/rpm/gratia-probe.spec) \
+    && yum-builddep -y gratia-probe/rpm/gratia-probe.spec \
+    && mkdir -p bld/upstream \
+    && echo "type=github repo=opensciencegrid/gratia-probe tag=master tarball=gratia-probe-$ver.tar.gz" > bld/upstream/gp.source \
+    && osg-build rpmbuild bld \
+    && yum install -y ./bld/_build_results/gratia-probe-*-*-*.rpm
```

Comment:
Can you split this up into different `RUN` invocations? Having them in a single layer means that rebuilding the image will force us to rebuild the entire image instead of only building the layers after the earliest change

Reply:
And here i thought it was an optimization to lump them all together!  :)

Sure, i can split it up.

How do you rebuild just the later layers?  (I don't actually know off hand)

Reply:
It should do it automatically if you've got a local copy of the image! It's more evident with things like `COPY` later in a Dockerfile. If you build it locally, then change the contents of the file that you're copying in, it'll only rebuiild the COPY + subsequent layers